webrtc/interfaces.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>IDL check of RTCPeerConnection</title>
```

Comment:
It tests all of https://w3c.github.io/webrtc-pc. Maybe just "WebRTC IDL tests" following https://github.com/w3c/web-platform-tests/blob/master/dom/interfaces.html